language_detector/main.py

```diff
@@ -5,5 +5,21 @@
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counter = {}
+    for diction in languages:
+        counter[diction['name']] = 0
+        for word in diction['common_words']:
+            for text_word in text.split():
+                if word == text_word:
+                    counter[diction['name']] += 1
```

Comment:
This is a really nice and concise solution all in 1 nested for loop. Our solution was a lot more verbose! We did it the other way round where for each word in the sample text, we ran through all the common words.